client/views/meet/CallPage.js

```diff
@@ -64,11 +65,42 @@ function CallPage({ roomId, visitorToken, visitorId, status, setStatus }) {
 
 	switch (status) {
 		case 'ringing':
-			// Todo Deepak
 			return (
-				<h1 style={{ color: 'white', textAlign: 'center', marginTop: 250 }}>
-					Waiting for the visitor to join ...
-				</h1>
+				<Flex.Container direction='column' justifyContent='center'>
+					<Box
+						width='full'
+						minHeight='sh'
+						textAlign='center'
+						backgroundColor='neutral-900'
+						overflow='hidden'
+						position='relative'
+					>
+						<Box
+							position='absolute'
+							zIndex='1'
+							style={{
+								top: '5%',
+								right: '2%',
+							}}
+							w='x200'
+						>
+							<div style={{ border: '1px solid black', padding: '30px' }}>
+								<UserAvatar username={agentName} className='rcx-message__avatar' size='x32' />
+							</div>
+						</Box>
+						<div style={{ marginTop: -180, padding: 20 }}>
+							<Box display='flex' align='center' justifyContent='center'>
+								<UserAvatar username={visitorName} className='rcx-message__avatar' size='x124' />
+							</Box>
+							<p style={{ color: 'white', fontSize: 15, textAlign: 'center', margin: 15 }}>
```

Comment:
I think we have a component named `Paragraph` for this in our codebase, so lets try to use it instead